### PR TITLE
Fixes #7927: Remove .cf files from initial promises

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/7927

### CODE REVIEW

**Comment by peckpeck** [comment]
Merge #1002 first

**Comment by peckpeck** [comment]
Big cleanup!

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
PR rebased

**Comment by peckpeck** [comment]
PR rebased

File: techniques/system/common/1.0/update.st [hunk 1]
```diff
@@ -155,8 +155,8 @@ bundle agent update_action
        # First copy rudder_promises_timestamp_tmp from the policy server, and check if it's newer
        # (based on the timestamp written in it) than rudder_promises_timestamp.
        # If so, copy all the promises
-       "rudder_promises_timestamp"     string => "rudder_promises_generated";
-       "rudder_promises_timestamp_tmp" string => "rudder_promises_generated.tmp";
+       "rudder_promises_timestamp"     string => "rudder_promises-generated";
+       "rudder_promises_timestamp_tmp" string => "rudder_promises-generated.tmp";
```
Comment (jooooooon) [inline techniques/system/common/1.0/update.st]: These two lines have a mix of _ and -

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**